sdk/keyvault/azure-security-keyvault-certificates/CMakeSettings.json

```diff
@@ -0,0 +1,210 @@
+﻿{
+  "configurations": [
+    {
+      "name": "x64-DebugWithTests",
+      "generator": "Ninja",
+      "configurationType": "Debug",

```

Comment:
is this VS exclusive?  Should we keep it ignored?

Reply:
it's in the root folder and not ignored there that's why i added it here

Reply:
Is there a way to use the one from the root folder everywhere?

Otherwise, we would have to maintain multiple files like this.. ( one per package )